decidim/decidim
# Add robots.txt instructions

#### :tophat: What? Why?
In this PR we add some `robots.txt` ruleset, that would prevent search engine crawling certain pages from the platform ( search and profiles). as requested [here](https://meta.decidim.org/processes/roadmap/f/122/proposals/16959?commentId=26319#comment_26319) 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10039
- Related to #10120

#### Testing
1. Make sure you run the `bundle exec rails decidim:robots:replace`
2. Check your development_app robots.txt for the following lines
```
# the following adds a rule for all bots to not index any page that contains a profile or search
User-agent: *
Disallow: /profiles/
Disallow: /search
```
3. Generate a new application & check the generated robots.txt file

### :camera: Screenshots

:hearts: Thank you!
